server/fixtures/articleValidationData.js

```diff
@@ -0,0 +1,17 @@
+
+export const emptyArticleData = {};
+
+export const invalidTitle = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
```

Comment:
Why is this title invalid ?

Reply:
The article title was set to not more than 100 characters and this title character is more than 100 characters.

Reply:
yeah more like 300